src/js/components/inspectors/Property.test.js

```diff
@@ -0,0 +1,91 @@
+/* eslint no-unused-expressions:0 */
+'use strict';
+
+var expect = require('chai').expect;
+var React = require('react');
+var enzyme = require('enzyme');
+var Rect = require('../../model/primitives/marks/Rect');
+var Property = require('./Property');
+var wrapper;
+
+describe('Property Inspector <Property/>', function() {
+  describe('Property Inspector <Property/ type="color"> (shallow)', function() {
```

Comment:
The slash in these describe block names should probably be moved to before the closing `>`